nextboot/nextboot.go

```diff
@@ -0,0 +1,147 @@
+package nextboot
+
+// Config contains a nextboot configuration for an ePoxy client.
+type Config struct {
+	V1 V1 `json:"v1,omitempty"`
```

Comment:
Please make sure that an empty or nonexistent V1 in the JSON doesn't crash things.